sw/device/tests/dif/dif_rv_timer_sanitytest.c

```diff
@@ -77,19 +77,29 @@ bool test_main(void) {
   uint64_t current_time;
   CHECK(dif_rv_timer_counter_read(&timer, kHart, &current_time) ==
         kDifRvTimerOk);
+
+  uint64_t timer_threshold = current_time + kDeadline;
   LOG_INFO("Current time: %d; timer theshold: %d", (uint32_t)current_time,
-           (uint32_t)(current_time + kDeadline));
+           (uint32_t)(timer_threshold));
   CHECK(dif_rv_timer_arm(&timer, kHart, kComparator,
                          current_time + kDeadline) == kDifRvTimerOk);
```

Comment:
If you're hoisting `timer_threshold`, propagate the use here, too.

Reply:
I've removed it.